### DISCUSSION TITLE
Deleted branch is shown at issue development

### DISCUSSION THREAD

**Discussion by acgc99** [community/community#70134] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi,
I'm starting to use branches on GitHub. I created a branch and assigned it to an issue develpment. After some time, I deleted the branch completely and asigned another branch.

The problem is that the old branch is still attached to the issue and when I try to access it, it is not found, and if I try to create a branch with the old name, it says it already exists.

The right branch is `issue-41`.

[Branch page](https://github.com/acgc99/Godot-Widgets/tree/41-simply-widgets-common-factors-better-naming-and-exported-properties) error 404.

![Issue](https://github.com/community/community/assets/96864661/9247de71-a793-45f0-b6ef-7ddb07de8801)

![Already exists](https://github.com/community/community/assets/96864661/27e94d67-0dfe-4c44-a651-ce837cfeb1ad)

![Branch list](https://github.com/community/community/assets/96864661/2d0b82cb-709b-466e-9751-2a187e43d635)

**Comment by MrMonkeyKing** [comment]
Currently experiencing the same issue. Sadly I have tried to delete the branch too before unlinking the issue. Could be a possible solution.
